### DISCUSSION TITLE
成功付款续期两年，账户有效期没有延长，没有收到验证码，手机现在也无法登录了

### DISCUSSION THREAD

**Discussion by GEHENNAhina42** [getlantern/lantern-forum-cn#592] [Q&A]
本人于2024年6月16日成功支付了蓝灯专业版续费2年服务，但已过16多小时邮箱至今未收到验证信息，现在手机无法登录，无法正常工作，请解决。邮箱为：867549807@qq.com)；付款成功截图附上
![Uploading 微信图片_20240617150205.jpg…
![微信图片_20240617150244](https://github.com/getlantern/lantern-forum-cn/assets/173033365/f647a895-bcbe-4945-a50a-da15d4cd58b5)
]()

**Comment by GEHENNAhina42** [comment]
![微信图片_20240617150244](https://github.com/getlantern/lantern-forum-cn/assets/173033365/98ca0b3f-718a-4f5d-a9b9-d0fd868755fe)

**Comment by MaxMKMax** [comment]
@GEHENNAhina42 您好. 好的, 我帮您提交问题. 请给我们几天工作日时间. 尽快回复您. 请您之后留意这个帖子或邮箱. 谢谢您的耐心等待! 也非常感谢您支持蓝灯!

**Comment by MaxMKMax** [comment]
@GEHENNAhina42 您好请问您的问题解决了吗?

**Reply by GEHENNAhina42** [reply]
没有解决，多次检查邮箱和垃圾箱都没有收到验证码，账户到现在你们也没有给我续期，手机还是无法登录，请快点解决问题并发送我的验证码
![Uploading 屏幕截图 2024-06-26 193539.png…]()

**Reply by GEHENNAhina42** [reply]
![屏幕截图 2024-06-26 193539](https://github.com/getlantern/lantern-forum-cn/assets/173033365/ed5dcc9c-0ba4-4008-b67b-e6006535ac4e)

**Reply by GEHENNAhina42** [reply]
从2017年使用到现在怎么会出现这种问题的？只是续期而已，付款这么久了还无法正常使用，要是找不到问题还无法续期请退款

**Comment by GEHENNAhina42** [comment]
没有解决，多次检查邮箱和垃圾箱都没有收到验证码，账户到现在你们也没有给我续期，手机还是无法登录，请快点解决问题并发送我的验证码


?露露姆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "getlantern/lantern-forum-cn"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月26日(星期三) 下午5:17
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [getlantern/lantern-forum-cn] 成功付款续期两年，账户有效期没有延长，没有收到验证码，手机现在也无法登录了 (Discussion #592)





 
@GEHENNAhina42 您好请问您的问题解决了吗?
